src/lexer.ts

```diff
@@ -134,7 +135,7 @@ export enum TokenType {
   Crossjoin = 'Crossjoin',
   AllResource = 'AllResource',
   ActionImportCall = 'ActionImportCall',
-  FunctionImportCall = 'FunctionImportCall',
+  // FunctionImportCall = 'FunctionImportCall',
```

Comment:
curious why comment function import call but not action import call

Reply:
We're referencing the `ActionImportCall` enum value within the `actionImportCall` function's return type, and that token type/value combination is used for the token returned there.  The `FunctionImportCall` enum value was not being referenced anywhere in the code & can safely be removed.  (I'll delete the commented code to make it clearer.)